lighthouse-service/main.go

```diff
@@ -38,7 +41,15 @@ type envConfig struct {
 }
 
 func main() {
+	kubeAPI, err := createKubeAPI()
+	if err != nil {
+		log.Fatalf("could not create kubernetes client: %s", err.Error())
+	}
+
 	var env envConfig
+	lighthouseService := LighthouseService{
+		KubeAPI: kubeAPI,
+	}
```

Comment:
```suggestion
	lighthouseService := LighthouseService{
		KubeAPI: kubeAPI,
	}
```
```suggestion
```
you can delete this from here and pass it dirctly as arg to _main below